base_layer/core/tests/chain_storage_tests/chain_backend.rs

```diff
@@ -1382,21 +1382,46 @@ fn fetch_target_difficulties<T: BlockchainBackend>(mut db: T) {
     header0.pow.pow_algo = PowAlgorithm::Blake;
     header0.pow.target_difficulty = Difficulty::from(100);
     let mut header1 = BlockHeader::from_previous(&header0);
-    header1.pow.pow_algo = PowAlgorithm::Monero;
-    header1.pow.target_difficulty = Difficulty::from(1000);
+    #[cfg(feature = "monero_merge_mining")]
```

Comment:
It would be somewhat cleaner to have 2 separate tests rather than all these `cfg()` directives

Reply:
Same comment for all modified tests

Reply:
Will separate the tests.